x/ibc/core/02-client/keeper/keeper.go

```diff
@@ -230,6 +230,13 @@ func (k Keeper) ValidateSelfClient(ctx sdk.Context, clientState exported.ClientS
 			tmClient.LatestHeight, ctx.BlockHeight())
 	}
 
+	// consensus params must match consensus params on executing chain
+	expectedConsensusParams := ctx.ConsensusParams()
```

Comment:
I double checked that these are set in baseapp before executing a message. see [`getContextForTx`](https://github.com/cosmos/cosmos-sdk/blob/c9cb02ea98a6d2b3387d996e46f4ca072c6b868c/baseapp/baseapp.go#L514)